### DISCUSSION TITLE
RadioGroup formControlLabelProps per Item

### DISCUSSION THREAD

**Discussion by Frozen-byte** [lookfirst/mui-rff#413] [Q&A]
**Is your feature request related to a problem? Please describe.**
I'd like to display a Radiogroup there the labels are facing in each side.
![image](https://user-images.githubusercontent.com/3965439/109749313-a10cf500-7bda-11eb-96bf-2163ea222e69.png)

This is currently not possible, because the `formControlLabelProps` are ignored in the `data` prop items.

**Describe the solution you'd like**
To keep backward compatibility, add a `formControlLabelProps` to the `data` items, those overwrite the parent props.

So this should work:
```
const RADIOS_DATA = [
  {
    label: "AAF",
    value: "aaf",
    formControlLabelProps: { labelPlacement: "start" },
  },
  {
    label: "CSAT",
    value: "csat",
    formControlLabelProps: { labelPlacement: "end" },
  },
];
render() {
    <Radios
      label="SWORD"
      name="fictionalSide"
      data={RADIOS_DATA}
    />
}
```

**Comment by lookfirst** [comment]
Moved to discussion since it isn't really an issue per se. I hope you don't mind.

This is an interesting concept for sure. I'd totally be open to accepting a PR around this. 

It shouldn't even be that hard... just add it to the RadioData interface and then add....

`{...data.formControlLabelProps}` after [this line](https://github.com/lookfirst/mui-rff/blob/master/src/Radios.tsx#L91) and it should override the parent props.

**Reply by Frozen-byte** [reply]
Yes it is that easy. I hacked it into my local copy for POC-reasons.
Was too lazy to make a proper MR tbh. 
Will create a PR soon(tm) when my Project goes into Alpha. 👍
